lib/assessment.rb

```diff
@@ -20,20 +27,39 @@
 # Name and location should be writeable, but age should be read-only.
 
 ##
-# your code here
+class Person
+  def initialize(name, age, location)
+    @name, @age, @location = name, age, location
+  end
+
+  def set_name(name)
+    @name = name
+  end
+
+  def set_location(location)
+    @location = location
+  end
```

Comment:
Although these methods will allow you to change the values of `name` and `location`, we were actually looking to see you use `attr_accessor` here.